### DISCUSSION TITLE
Possible bug with time only Date Picker

### DISCUSSION THREAD

**Discussion by kellis5137** [Budibase/budibase#8916] [Help] [answered]
I'm finding a weird issue with the Date picker. If you choose to have a "time only" picker, it seems the picker automatically advances the time by 1 hour when you click into it, then click out of it. Or if you click into it, advance the time by 1 hour, then step out... it goes up one. A user actually found this issue.

Screen shots in order...
<img width="1412" alt="Screenshot 2022-12-02 at 7 39 42 PM" src="https://user-images.githubusercontent.com/1899734/205414018-7e1b0c0d-e0e5-4906-ba07-6cf74fe19361.png">
<img width="1388" alt="Screenshot 2022-12-02 at 7 39 54 PM" src="https://user-images.githubusercontent.com/1899734/205414021-eb122f55-e9c7-477e-a3b6-ef7e593d776e.png">
<img width="1406" alt="Screenshot 2022-12-02 at 7 40 05 PM" src="https://user-images.githubusercontent.com/1899734/205414025-ee3eda5a-c098-48ab-860b-d01b1cb8ca1c.png">

Keiths app-export-1670027839074.tar.gz

**Comment by kellis5137** [comment]
[Keiths app-export-1670027839074.tar.gz](https://github.com/Budibase/budibase/files/10145236/Keiths.app-export-1670027839074.tar.gz)

**Comment by michal-81** [comment]
I have noticed that too. 
As a workaround try to to insert some random text as a default value for the time picker. It helped in my case.
I have also noticed that when saving row the actual time stored in db (MySQL in my case) is shifted by minus 1 hour even though ignore timezone setting is checked so in 'Save Row' action I use code like this to fix it.

```
if ($("Form.Fields.start_time")){
    var dt = new Date($("Form.Fields.start_time"));
    dt.setMinutes(dt.getMinutes() + 60);
    return dt;
}

```

**Comment by punkshift** [comment]
I've noticed this too. It's minor yet annoying. lol
@mjashanks

**Comment by melohagan** [comment, marked as answer]
This is a bit annoying. I've raised an issue here: https://github.com/Budibase/budibase/issues/9138
